packages/commercetools/theme/components/Checkout/CartPreview.vue

```diff
@@ -74,7 +99,7 @@ import {
   SfInput,
   SfCircleIcon
 } from '@storefront-ui/vue';
-import { computed, ref } from '@nuxtjs/composition-api';
+import { computed, ref } from '@vue/composition-api';
```

Comment:
why replacing the `nuxtjs/composition-api`?? We are not using `vue/composition-api` anymore

Reply:
@alefbarbeli I was copying the component's code from another repo and this got through by accident. Reverted [here](https://github.com/vuestorefront/vue-storefront/pull/6518/commits/56d44a8706d914edd54aec8e0f590096219883db)